doc/documentation_contributors.md

```diff
@@ -0,0 +1,93 @@
+Guideline for contributing documentation
+========================================
+
+The DLite documentation is written in [Markdown].
+This include both the README files and documentation found in the `doc/` subdirectory.
+
+Common to both is that the text should be as easy and natural as possible to read and write both from the terminal, in an editor and rendered in a web browser.
+Hence, the following recommendations:
+
+* Write one sentence per line, in order to get an easier to read output from `git diff`.
+  This improves readability and understandability when performing reviews on GitHub.
+
+* For README files, use the underline ([setext]) style format for main and sub-headers.
+  For example:
+
+      Header level 1
+      ==============
+      ...
+
+      Header level 2
+      --------------
+      ...
+
+  For documentation, follow the style used by the original contributor.
+
+* Avoid the use of raw HTML.
+
+* Links become more readable if you place them at the end of the document using square brackets.
+  Example:
+
+```markdown
+A link to [SOFT].
+...
+
+End of document.
+
+
+[SOFT]: https://www.sintef.no/en/publications/publication/1553408/
+```
+
+* Use `.md` as file extension for all Markdown files.
+
+The README files are intended to document the overall project or the content in a subdirectory.
+These should therefore render nicely on GitHub.
+Hence, use [CommonMark] or possible [GitHub-flavored Markdown].
```

Comment:
```suggestion
Hence, use [CommonMark] or [GitHub-flavored Markdown].
```

Reply:
CommonMark is preferred, since we it is not guaranteed that we will be using github forever.